### DISCUSSION TITLE
Wake model constants tuned for offshore or onshore?

### DISCUSSION THREAD

**Discussion by Ana-GS** [NatLabRockies/floris#459] [Q&A]
Dear Floris team,

I would like to ask you if the wake model constants you use as defaults in your example files (cc.yaml, gch.yaml, jensen.yaml) have been tuned to offshore or onshore projects. I am using Floris 3.0.

Thanks!

Kind regards,

Ana

**Comment by paulf81** [comment]
hi Ana, I am sorry for missing this question.  Generally speaking the tunings for the different models are matched to SOWFA simulations of offshore farms, however changing the value of the turbulence intensity can be used to match better toward onshore farms.

More specifically you can see tuning of the GCH model to offshore SCADA data in this paper:
https://www.mdpi.com/1996-1073/15/6/1964 (Table A.1)

And cumulative curl model in this paper:
https://wes.copernicus.org/preprints/wes-2022-17/
